go/client/ui.go

```diff
@@ -717,7 +717,7 @@ func (ui SecretUI) GetPassphrase(pin keybase1.GUIEntryArg, term *keybase1.Secret
 	if term == nil {
 		term = &keybase1.SecretEntryArg{
 			Prompt:         pin.Prompt,
-			UseSecretStore: pin.Features.SecretStorage.Allow,
```

Comment:
let's make the names match so we can have a more generalized algorithm for features